### PR TITLE
Remove tagging from interface

### PR DESCRIPTION
**Pull request by jackscotti**
This commit removes the tagging interface for Topics (SpecialistSectors), MainstreamBrowsePages(Sections), Organisations.
Tagging will be handled from the ContentTagger application.

Ticket: https://trello.com/c/4vU2VrC0/598-move-remaining-tagging-from-panopticon-to-content-tagger
Blocked by: https://trello.com/c/qXmMfTOZ/608-confirm-that-links-in-publishing-api-are-accurate-complete

### CODE REVIEW

File: app/views/artefacts/form/_tags.html.erb [hunk 1]
```diff
@@ -1,56 +1,5 @@
 <%- unless f.object.app_without_tagging_support? %>
   <%- if f.object.tagging_migrated? %>
```
Comment (tijmenb) [inline app/views/artefacts/form/_tags.html.erb]: All the things will be "migrated" after this, so this conditional can be removed.

**Comment by tijmenb** [comment]
🔥

**Comment by tijmenb** [comment]
https://github.com/alphagov/panopticon/blob/master/config/settings/test.yml can go too.

**Comment by tijmenb** [comment]
Did you check for CSS? There might be some stuff there.

**Comment by tijmenb** [comment]
You can also update https://github.com/alphagov/panopticon/blob/master/README.md#features--deprecation (`Tagging documents`).

File: app/views/artefacts/form/_tags.html.erb (hunk 1 above)
Comment (jackscotti) [inline app/views/artefacts/form/_tags.html.erb]: gotcha. So should I remove any reference to 'migrated apps' such as: https://github.com/alphagov/panopticon/search?utf8=%E2%9C%93&q=migrated&type=Code

Comment (tijmenb) [inline app/views/artefacts/form/_tags.html.erb]: Yeah. Perhaps you can replace the entire contents of this template with the message from `migrated_app.html.erb`.

**Comment by jackscotti** [comment]
@tijmenb updated. 
🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥

**Comment by mobaig** [comment]
Comment on https://github.com/alphagov/panopticon/pull/360/commits/aaf43ed5045b61373b18a03ade7ce3e18c45a39a reads: `Publisher is an app that has exceptions for this reason should not be used as a default.`

I'm not sure I understand. Can you clarify?

**Comment by mobaig** [comment]
Apart from a commit message that I think could do with re-writing, this looks good to me. 👍

**Comment by jackscotti** [comment]
@mobaig does the description of https://github.com/alphagov/panopticon/pull/360/commits/91231a857410e69d2a30f6c824dd495eedcfbe2c make sense now?

**Comment by jackscotti** [comment]
Closing this for now. Got the 👍  from @mobaig IRL.

**Pull request merged**